vlab-research/fly
# Feature/ratings opinion component

### What changes have you made?
- Swapped in a bigger sample set to test more field type variations
- Created a shared component for fields of type `opinion_scale` and `rating`
- Set up dynamic component rendering from the form component

### Which issue(s) does this PR address?
#88 

### How to test
- Run npm install and open up the web survey on `localhost:5000`
- Navigate through the survey and check that you're able to submit an answer to either an `opinion_scale` or `rating` question
